### PR TITLE
Commit #1. velocity_of_liquid_flowing_out_of_pipe

### PR DESCRIPTION
**Pull request by Zulusssss**
Add law and tests.

### CODE REVIEW

File: symplyphysics/laws/hydro/velocity_of_liquid_flowing_out_of_pipuid_flowing_out_of_pipe.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+from sympy import (Eq, solve, sqrt)
```
Comment (blackyblack) [inline symplyphysics/laws/hydro/velocity_of_liquid_flowing_out_of_pipuid_flowing_out_of_pipe.py:1]: Fix file name. Add to file name, what do we use to calculate velocity.

File: symplyphysics/laws/hydro/velocity_of_liquid_flowing_out_of_pipuid_flowing_out_of_pipe.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+## v - velocity of the liquid flowing out of the pipe,
+## p - pressure,
+## p0 - density of the liquid.
+
+velocity = Symbol("velocity", units.velocity)
+
+pressure = Symbol("pressure", units.pressure)
+density = Symbol("density", units.mass / units.volume)
+
+law = Eq(velocity, sqrt( 2 * pressure / density))
```
Comment (blackyblack) [inline symplyphysics/laws/hydro/velocity_of_liquid_flowing_out_of_pipuid_flowing_out_of_pipe.py:25]:
```suggestion
law = Eq(velocity, sqrt(2 * pressure / density))
```

Comment (blackyblack) [inline symplyphysics/laws/hydro/velocity_of_liquid_flowing_out_of_pipuid_flowing_out_of_pipe.py:25]: Add proof using https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/hydro/velocity_from_height.py

File: test/hydro/velocity_of_liquid_flowing_out_of_pipuid_flowing_out_of_pipe_test.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+@fixture(name="test_args")
+def test_args_fixture():
+    pressure = Quantity(500e3 * units.pascal)
+    density = Quantity(997 * (units.kilogram / units.meter**3))
+
+    Args = namedtuple("Args", ["pressure", "density"])
+    return Args(pressure=pressure, density=density)
+
+
+def test_basic_electric_dipole_moment(test_args):
```
Comment (blackyblack) [inline test/hydro/velocity_of_liquid_flowing_out_of_pipuid_flowing_out_of_pipe_test.py:21]: Wrong name.

File: symplyphysics/laws/hydro/velocity_of_liquid_flowing_out_of_pipuid_flowing_out_of_pipe.py (hunk 2 above)
Comment (Zulusssss) [reply symplyphysics/laws/hydro/velocity_of_liquid_flowing_out_of_pipuid_flowing_out_of_pipe.py:25]: > Add proof using https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/hydro/velocity_from_height.py

As I understand it, my law cannot be derived from the above.
The law "velocity_from_height" is derived from Bernoulli's law for a special case (the body is filled with liquid and there is a hole in the body).
In the same law, we are talking about a tube located horizontally. I can get the velocity from Bernoulli's equation, but it is not among the laws.

**Review by blackyblack — APPROVED** [review]

**Pull request merged**